### ISSUE TITLE
Some rules don't print their identifiers when reporting

### ISSUE THREAD

**Issue by ZevEisenberg** [realm/SwiftLint#180]
I have code like this:

```swift
if theta > π {
    value = twoPi - theta
}
else {
    value = theta
}
```
I'm getting this warning:
> Statement Position Violation: Else and catch must be on the same line and one space after previous declaration ()

There is no rule indicated in the parentheses, which seems to indicate that there's no way to turn this off. Is that correct?

**Comment by jpsim** [comment]
It can be turned off. That rule's identifier is `statement_position`. I'll look into why the identifier isn't being printed.
